nodejs/package.json

```diff
@@ -11,7 +11,10 @@
   "scripts": {
     "build": "npm install && mkdir -p node_modules/newrelic-lambda-wrapper && cp index.js node_modules/newrelic-lambda-wrapper && mkdir -p nodejs && cp -r node_modules nodejs && zip -rq ../dist/nodejs/NewRelicLayer.zip nodejs && rm -rf ./nodejs",
     "clean": "rm ../dist/nodejs/NewRelicLayer.zip",
-    "lint": "eslint ./*.js"
+    "lint": "eslint ./*.js",
```

Comment:
this is fine now but since our team maintains this maybe we use the `@newrelic/eslint-config` ruleset later